src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,30 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateStart = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateFinish = LocalDate.parse(dateTo, formatter);
+        StringBuilder result = new StringBuilder("Report for period ").append(dateFrom)
+                .append(" - ").append(dateTo).append(System.lineSeparator());
```

Comment:
Create a constant for " - " and reuse it 
```suggestion
                .append(" - ").append(dateTo));
```